src/Tizen.Security.TEEC/Tizen.Security.TEEC/Libteec.cs

```diff
@@ -597,6 +600,7 @@ internal void Open64(Guid destination, uint loginMethod, byte[] connectionData,
         /// <exception cref="InvalidOperationException">The operation is invalid.</exception>
         public void Close() {
```

Comment:
How about check closed before CloseSession?

Reply:
I think the implementation should behave correctly. This is C# wrapper only.
I have put a call to Close() in destructor, to prevent resource leakage and this special call have to be guarded but when programmer calls it double it should be forwarded to implementation.

Documentation says:
*This function closes a Session which has been opened with a Trusted Application.
All Commands within the Session MUST have completed before calling this function.
The Implementation MUST do nothing if the session parameter is NULL.
The implementation of this function MUST NOT be able to fail: after this function returns the Client
Application must be able to consider that the Session has been closed.*

Reply:
To prevent leakage of unmanaged resource, 
Implement IDisposable interface

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/dispose-pattern

Reply:
Done